src/controllers/task.list.controller.ts

```diff
@@ -27,10 +28,13 @@ export const get = async (req: Request, res: Response, next: NextFunction) => {
     const taskList: TaskList = initTaskList(company.companyNumber, transactionId, submissionId, confirmationStatement);
     taskList.recordDate = calculateFilingDate(taskList.recordDate, company);
 
+    taskList.allTasksCompleted = Object.keys(taskList.tasks).length === taskList.tasksCompletedCount;
```

Comment:
can you move the setting of all tasks completed into the initTaskList function in task list service?